src/common/currencyUtil.ts

```diff
@@ -1,21 +1,47 @@
+const SATOSHIS_PER_COIN = 10 ** 8;
+
+const CURRENCY_DICTIONARY: {
+  [key: string]: { unitsPerSatoshi: number; fullName?: string };
+} = {
+  BTC: { unitsPerSatoshi: 1, fullName: "Bitcoin" },
```

Comment:
Ugh, I really hate that we have to keep the satoshis per coin hardcoded here. At the same time I don't have a better solution as long as we use satoshis from coin from xud side.

Let's not address it in this PR, but keep this in mind in the future.

Reply:
The only real solution on xud side is https://github.com/ExchangeUnion/xud/issues/1797 , right?

Reply:
As it is currently not used as we are not yet calculating anything down to the smallest unit, it can be removed for now and when it becomes an actual need, let's implement it according to whatever is the current state of things.
I at first thought we might need this data on the UI side but in the end it came out it's not necessary at the moment. Forgot to remove it.

Reply:
Something like that, yes.